package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "cerestats",
-  "version": "0.20.0",
+  "version": "0.21.0",
```

Comment:
Update all `package.json` files. You can find some in the directories: api, backend, etc.

Reply:
fixed